jsonrpc/src/schemas/trace.json

```
@@ -0,0 +1,89 @@
+{
+  "traceResultObject": {
+    "title": "traceResultObject",
+    "description": "Trace data for the result of tracing content request.",
+    "type": "object",
+    "required": [
+      "origin",
+      "targetId",
+      "receivedFrom",
+      "responses",
+      "metadata"
+    ],
+    "properties": {
+      "origin": {
+        "description": "Local Node ID",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "targetId": {
+        "description": "Target content ID",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "receivedFrom": {
+        "description": "Node ID from which the content was received.",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "responses": {
+        "$ref": "#/components/schemas/traceResultResponses"
+      },
+      "metadata": {
+        "$ref": "#/components/schemas/traceResultMetadata"
+      },
+      "cancelled": {
+        "$ref": "#/components/schemas/listOfNodeIds"
+      }
+    }
+  },
+  "listOfNodeIds": {
+    "title": "listOfNodeIds",
+    "description": "Contains list of nodes IDs.",
+    "type": "array",
+    "items": {
+      "$ref": "#/components/schemas/uint256"
+    }
+  },
+  "traceResultResponses": {
+    "title": "traceResultResponses",
+    "description": "Contains a map from remote node ID, to the node IDs it responded with. For the Node ID that is also in the `receivedFrom` field, respondedWith must be empty array.",
+    "type": "object",
+    "additionalProperties": {
+      "$ref": "#/components/schemas/traceResultResponseItem"
+    }
+  },
+  "traceResultResponseItem": {
+    "title": "traceResultResponseItem",
+    "description": "Contains information about nodes' responses, including the duration form the begining of the original request up to the moment of receiving this response.",
+    "type": "object",
+    "properties" : {
+      "durationsMs": {
+        "description": "Time it took from the begining of the original request up to receiving this response.",
```

```suggestion
        "description": "Time it took from the beginning of the original request up to receiving this response.",
```

Original request meaning the individual request or the start of the lookup?